index.html

```diff
@@ -110,72 +112,182 @@
             };
         }
 
-		$(document).ready(function(){
-			
+        /* Set up TinCanJS */
+        var tincan = new TinCan (
+            {
+                recordStores: [
+                    {
+                        endpoint: Config.endpoint,
+                        username: Config.authUser,
+                        password: Config.authPassword,
+                        allowFail: false
+                    }
+                ]
+            }
+        ),
+        PrototypesLauncher = {};
+        PrototypesLauncher.Activity = {
+            id: "http://id.tincanapi.com/activity/tincan-prototypes/launcher",
+            definition: {
+                type: "http://id.tincanapi.com/activitytype/lms",
+                name: {
+                    "en-US": "Tin Can Prototypes Launcher"
+                },
+                description: {
+                    "en-US": "A tool for launching the Tin Can prototypes. Simulates the role of an LMS in launching experiences."
+                }
+            }
+        };
+        PrototypesLauncher.context = {
+            contextActivities : {
+                grouping: [
+                    {
+                        id: "http://id.tincanapi.com/activity/tincan-prototypes"
+                    }
+                ],
+                category: [
+                     {
+                        id: "http://id.tincanapi.com/recipe/tincan-prototypes/launcher/1",
+                        definition: {
+                            type: "http://id.tincanapi.com/activitytype/recipe"
+                        }
+                    },
+                    {
+                        id: "http://id.tincanapi.com/activity/tincan-prototypes/launcher-template",
+                        definition: {
+                            type: "http://id.tincanapi.com/activitytype/source",
+                            name: {
+                                "en-US": "Tin Can Launcher Template"
+                            },
+                            description: {
+                                "en-US": "A launch tool based on the Tin Can launch prototypes."
+                            }
+                        }
+                    }
+                ]
+            }
+        };
+
+        tincan.sendStatement({  
```

Comment:
If we don't care about whether the statement succeeded or not, it can be as simple as providing the callback argument and an empty function. The key is that we don't want to block waiting for a statement to send when we likely don't care if it succeeded, IOW, the callback can be a no-op that just causes the request handling under the hood to be async.